README.md

```diff
@@ -1 +1,44 @@
-# Portfolio1
\ No newline at end of file
+# Portfolio
+
+> This is my first attempt at making my portfolio. 
+
+
+This is my firt attempt at portfolio built with flex boxes.
+I have added a toolbar and the headline section.
+Some button interactions like hover were implemented.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+- Create a new repo
+- Add a feature branch to the repo
+- Add linters
+- Add a git ignore file
+- Add HTML & CSS files
+- Open a pull request
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@chaw-bot](https://github.com/chaw-bot)
+- Twitter: [@chaw36422087](https://twitter.com/chaw36422087)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.

```

Comment:
- [ ] **OPTIONAL**
Please make sure to fix the MIT link because it's broken. Keep in mind that such a very small mistake might leave a very bad impression on future viewers. It's so important for your projects to look professional.

> Please note that this is a mandatory requirement, and I'm sorry that it didn't show up for you in the earlier review.